flow/model/qvalue/avro_converter.go

```diff
@@ -219,7 +219,7 @@ func QValueToAvro(
 	case types.QValueTime:
 		return c.processNullableUnion(c.processGoTime(v.Val))
 	case types.QValueTimeTZ:
-		return c.processNullableUnion(c.processGoTimeTZ(v.Val))
+		return c.processNullableUnion(c.processGoTime(v.Val))
```

Comment:
timezone?

Reply:
Timezone is never maintained. Postgres is responsible for collapsing timezone info now (which actually works, so that you get the UTC equivalent of the timetz value, rather than simply dropping timezone info, which breaks if the source has heterogeneous timezones on their values)